app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,31 @@
+class Admin::OrdersController < Admin::BaseController
+  def index
+    @orders = Order.all
+    @users = User.all
+    if params[:email].present?
+      @users = [User.find_by(email: params[:email])]
+      @orders = @users.first.orders
+    end
+  end
+
+  def mark_status
+    order_transaction = OrderTransaction.find_by(order_id: params[:order_id])
+    order = Order.find(params[:order_id])
```

Comment:
before_action

Reply:
Extacted it in before_action named `order_transaction_for-order_in_params`